lib/terraforming/resource/efs.rb

```diff
@@ -0,0 +1,62 @@
+# author: Jim Conner <snafu.x@gmail.com>
+# Nov 2016
+module Terraforming
+  module Resource
+    class EFS
```

Comment:
Please match Ruby class name to target Terraform resource name.
In this case, class name should be `EFSFileSystem`.

## REF

https://www.terraform.io/docs/providers/aws/r/efs_file_system.html